cli/view.js

```diff
@@ -74,28 +75,39 @@ const loadAndAddHandlers = ({app, handlersArg, datasetDir, narrativeDir}) => {
     `Looking for datasets in ${datasetsPath}\nLooking for narratives in ${narrativesPath}`;
 };
 
-const getAuspiceBuild = () => {
+const getAuspiceBuild = (customBuildOnly) => {
   const cwd = path.resolve(process.cwd());
   const sourceDir = path.resolve(__dirname, "..");
-  if (
-    cwd !== sourceDir &&
-    fs.existsSync(path.join(cwd, "index.html")) &&
-    fs.existsSync(path.join(cwd, "dist")) &&
-    fs.readdirSync(path.join(cwd, "dist")).filter((fn) => fn.match(/^auspice.bundle.[a-z0-9]+.js$/)).length === 1
-  ) {
-    return {
-      message: "Serving the auspice build which exists in this directory.",
-      baseDir: cwd,
-      distDir: path.join(cwd, "dist")
-    };
+
+  // Default to current working directory.
+  let baseDir = cwd;
+  if (!hasAuspiceBuild(cwd)) {
+    if (cwd === sourceDir || customBuildOnly) {
+      utils.error(`Auspice build files not found under ${cwd}. Did you run \`auspice build\` in this directory?`);
+      process.exit(1);
+    } else if (!hasAuspiceBuild(sourceDir)) {
+      utils.error(`Auspice build files not found under ${cwd} or ${sourceDir}. Did you run \`auspice build\` in either directory?`);
+      process.exit(1);
+    } else if (cwd !== sourceDir) {
```

Comment:
This conditional will always be true (if cwd is the sourceDir, then the if statement on line 85 will be true). I suggest this be changed to

```js
if (!hasAuspiceBuild(cwd)) {
    if (cwd === sourceDir || customBuildOnly) {
      utils.error(`Auspice build files not found under ${cwd}. Did you run \`auspice build\` in this directory?`);
      process.exit(1);
    } else if (!hasAuspiceBuild(sourceDir)) {
      utils.error(`Auspice build files not found under ${cwd} or ${sourceDir}. Did you run \`auspice build\` in either directory?`);
      process.exit(1);
    }
   utils.log(`Auspice build files not found under ${cwd}. Using build files under ${sourceDir}.`)
   baseDir = sourceDir;
}
```

I also wouldn't use an `else if` after a preceding return / exit call, but I think this is probably a stylistic thing.

Reply:
Updated with your suggestions in 1d2a023e22004df7492e2cc9b7847c941ec8efeb.